1-js/02-first-steps/15-function-expressions/article.md

```diff
@@ -148,143 +148,143 @@ function ask(question, yes, no) {
 
 *!*
 ask(
-  "Do you agree?",
-  function() { alert("You agreed."); },
-  function() { alert("You canceled the execution."); }
+  "Zgadzasz się?",
+  function() { alert("Może być."); },
+  function() { alert("Nie chcesz nic wywoływać."); }
 );
 */!*
 ```
 
-Here, functions are declared right inside the `ask(...)` call. They have no name, and so are called *anonymous*. Such functions are not accessible outside of `ask` (because they are not assigned to variables), but that's just what we want here.
+W tym przykładzie, funkcje deklarowane są bezpośrednio wewnątrz wywołania funkcji `ask(...)`. Nie są w żaden sposób nazwane, dlatego nazywa się je *anomimowymi*. Takie funkcje nie są dostępne poza wywołaniem `ask` (ponieważ nie są przypisane do zmiennych), jednak w tym przypadku jest to dokładnie to, czego chcemy.
 
-Such code appears in our scripts very naturally, it's in the spirit of JavaScript.
+Taki kod będzie pojawiał się w naszych skryptach naturalnie - jest to część filozofii JavaScriptu.
 
-```smart header="A function is a value representing an \"action\""
-Regular values like strings or numbers represent the *data*.
+```smart header="Funkcja to zmienna reprezentująca \"czynność\""
+Zwykłe zmienne, jak liczby, czy łańcuchy znaków, reprezentują *dane*.
 
-A function can be perceived as an *action*.
+O funkcji można myśleć jak o *działaniu*.
 
-We can pass it between variables and run when we want.
+Możemy przekazywać ją pomiędzy zmiennymi i wywoływać kiedy chcemy.
 ```
 
 
-## Function Expression vs Function Declaration
+## Wyrażenie funkcyjne a deklaracja funkcji
 
-Let's formulate the key differences between Function Declarations and Expressions.
+Podsumujmy najważniejsze różnice między deklaracją funkcji a wyrażeniem funkcyjnym.
 
-First, the syntax: how to differentiate between them in the code.
+Po pierwsze, składnia - jak odróżnić je w kodzie.
 
-- *Function Declaration:* a function, declared as a separate statement, in the main code flow.
+- *Deklaracja funkcji:* funkcja, zadeklarowana jako osobne polecenie, w głównym ciągu kodu.
 
     ```js
-    // Function Declaration
+    // deklaracja funkcji
     function sum(a, b) {
       return a + b;
     }
     ```
-- *Function Expression:* a function, created inside an expression or inside another syntax construct. Here, the function is created at the right side of the "assignment expression" `=`:
+- *Wyrażenie funkcyjne:* funkcja, utworzona wewnątrz wyrażenia lub innej konstrukcji składniowej. Poniżej, funkcja jest tworzona po prawej stronie "wyrażenia przypisania" `=`:
 
     ```js
-    // Function Expression
+    // wyrażenie funkcyjne
     let sum = function(a, b) {
       return a + b;
     };
     ```
 
-The more subtle difference is *when* a function is created by the JavaScript engine.
+Bardziej subtelną różnicą jest różnica w *momencie tworzenia* funkcji przez silnik JavaScriptu.
```

Comment:
Pleace replace `Bardziej subtelną różnicą jest różnica w *momencie tworzenia* funkcji przez silnik JavaScriptu.` by `Bardziej subtelną różnicą jest to *kiedy* funkcja jest tworzona przez silnik JavaScriptu.`